pkgs/development/compilers/julia/update-1.5.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i python3 -p python3 python3Packages.requests
+
+import os
+import re
+import requests
+import subprocess
+
+latest = requests.get("https://api.github.com/repos/JuliaLang/julia/releases/latest").json()["tag_name"]
+assert latest[0] == "v"
+major, minor, patch = latest[1:].split(".")
+assert major == "1"
+# When a new minor version comes out we'll have to refactor/copy this update script.
```

Comment:
What do you mean by this?  I see "1.5.nix" hard-coded below, but otherwise things should work, no?

Reply:
Yeah, this is only meant to indicate that a future 1.x version may have a completely different .nix file which may be structured differently. Realistically there should likely be enough overlap for the same logic to be reused. The asserts are meant to be for defensive coding.